test/e2e-helm/e2e_suite.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
Remove this file for now. In a follow-up, we can port the shell-based test to a Go-based test and reintroduce this when it covers all of the existing test scenarios.